.github/workflows/main.yml

```diff
@@ -0,0 +1,135 @@
+---
+name: ci
+on:
+  push:
+    branches:
+      - main
+      - setup-beam
```

Comment:
I will remove this once ready to merge, for now it's useful to see the workflow in action : 

https://github.com/starbelly/elixir-mllp/actions/runs/1441478368